tests/mb_enhanced_cover_art_uploads/providers/vgmdb.test.ts

```diff
@@ -96,4 +97,133 @@ describe('vgmdb provider', () => {
                 .toStrictEqual(expected);
         });
     });
+
+    describe('checking URL support', () => {
+        it('matches album URLs', () => {
+            expect(provider.supportsUrl(new URL('https://vgmdb.net/album/79')))
+                .toBeTrue();
+        });
+
+        it.each`
+            url | type
+            ${'https://vgmdb.net/artist/77'} | ${'artist'}
+            ${'https://vgmdb.net/org/186'} | ${'organisation'}
+        `('does not match artist $type URLs', ({ url }: { url: string }) => {
+            expect(provider.supportsUrl(new URL(url)))
+                .toBeFalse();
+        });
+    });
+
+    describe('finding images', () => {
+        it('finds all images if they are all public', async () => {
+            const covers = await provider.findImages(new URL('https://vgmdb.net/album/96418'));
+
+            expect(covers).toBeArrayOfSize(2);
+            expect(covers[0].url.pathname).toBe('/albums/81/96418/96418-1581893265.jpg');
+            expect(covers[0].type).toStrictEqual([ArtworkTypeIDs.Front]);
+            expect(covers[0].comment).toBeEmpty();
+            expect(covers[1].url.pathname).toBe('/albums/81/96418/96418-1581893266.jpg');
+            expect(covers[1].type).toStrictEqual([ArtworkTypeIDs.Back]);
+            expect(covers[1].comment).toBeEmpty();
+        });
+
+        it('does not find all images if some are not public', async () => {
+            // This may seem like a useless test case, but if it starts working
+            // all of a sudden, I'd like to know about it
+            const covers = await provider.findImages(new URL('https://vgmdb.net/album/79'));
```

Comment:
How is this supposed to return a different number of covers in the future if you have the response cached with polly? Or did I misunderstand how this works?

Reply:
We'll periodically run these tests in CI in polly's passthrough mode, in which case it'll make actual requests. Added a clarification comment